### PR TITLE
Airspace Lookup button

### PR DESCRIPTION
**Pull request by AlphaLima**
new Meneu button in NAV 3/3 Menue for the new Airspace Lookup dialog.

### CODE REVIEW

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp [hunk 1]
```diff
@@ -320,7 +320,10 @@ bool CustomKeyHandler(const int key) {
 		PlayResource(TEXT("IDR_WAV_CLICK"));
 		InputEvents::eventWaypointDetails(_T("select"));
 		return true;
-
+    case ckAirspaceLookup:
```
Comment (brunotl) [inline Common/Source/LKInterface/LKCustomKeyHandler.cpp]: case must be ordered

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp [hunk 2]
```diff
@@ -320,7 +320,10 @@ bool CustomKeyHandler(const int key) {
...
+        PlayResource(TEXT("IDR_WAV_CLICK"));
```
Comment (brunotl) [inline Common/Source/LKInterface/LKCustomKeyHandler.cpp]: why not the same sound ? cf. InputsEvents.cpp line 2019

File: Common/Source/InputEvents.cpp [hunk 3]
```diff
@@ -2105,6 +2105,13 @@ void InputEvents::eventService(const TCHAR *misc) {
 	return;
   }
 
+  if (_tcscmp(misc, TEXT("AIRSPACELU")) == 0) {
+      LKSound(TEXT("LK_KLICK.WAV"));
+       dlgAirspaceSelect();
+        return;
+  }
+
```
Comment (brunotl) [inline Common/Source/InputEvents.cpp]: why not use CustomKeyHandler ? 
like that you avoid to have two different code for same feature ....1

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp (hunk 1 above)
Comment (brunotl) [inline Common/Source/LKInterface/LKCustomKeyHandler.cpp]: case must be ordered

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp (hunk 2 above)
Comment (brunotl) [inline Common/Source/LKInterface/LKCustomKeyHandler.cpp]: why not the same sound ? cf. InputsEvents.cpp line 2019

File: Common/Source/InputEvents.cpp (hunk 3 above)
Comment (brunotl) [inline Common/Source/InputEvents.cpp]: why not use CustomKeyHandler ?
like that you avoid to have two different code for same feature ....1

File: Common/Header/LKInterface.h [hunk 4]
```diff
@@ -103,6 +103,7 @@ typedef enum{
 	ckResetComm,
 	ckDspMode,
 	ckOtimizerPointsToggle,
+	ckAirspaceLookup,
```
Comment (brunotl) [inline Common/Header/LKInterface.h]: if you add CustomKey, you need also add CustomKeyLabel

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp (hunk 1 above)
Comment (AlphaLima) [reply Common/Source/LKInterface/LKCustomKeyHandler.cpp]: ???

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp (hunk 2 above)
Comment (AlphaLima) [reply Common/Source/LKInterface/LKCustomKeyHandler.cpp]: Just copied it from the case above, why make it different than other menue entries?

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp (hunk 1 above)
Comment (brunotl) [reply Common/Source/LKInterface/LKCustomKeyHandler.cpp]: writing

switch(..) {
case 1:
case 2:
case 3:
}

are better than writing

switch(..) {
case 3:
case 1:
case 2:
}

especially when all cases are contiguous value

File: Common/Source/LKInterface/LKCustomKeyHandler.cpp (hunk 2 above)
Comment (brunotl) [reply Common/Source/LKInterface/LKCustomKeyHandler.cpp]: ok, here it's not menu entrie, it's custom key... 
so if i configure custom with this, i don't have same sound than standart menu you have added ...

after that, agree, this is probably the right sound, for avoid this problem, you can change your change in InputEvents.cpp using same way used for "SONAR" or "OPTIMODE"

File: Common/Header/LKInterface.h (hunk 4 above)
Comment (AlphaLima) [reply Common/Header/LKInterface.h]: done!

**Pull request merged**